microfrontends/SignDocuments/package.json

```diff
@@ -32,6 +32,7 @@
     "react-scrollbars-custom": "^4.1.1",
     "react-signature-canvas": "^1.0.6",
     "reactour": "^1.19.1",
+    "select-dom": "^9.0.0",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
```

Comment:
Is there a specific reason for using extra select-dom package.

Reply:
Only as an helper class with lightweight selection.

Reply:
Hello, @rishabjasrotia, can you please help us out to resolve conflict?

Reply:
Yes sure I will resolve conflicts